AzureBot/Dialogs/ActionDialog.cs

```diff
@@ -193,6 +193,43 @@ public class ActionDialog : LuisDialog<string>
             await this.ProcessVirtualMachineActionAsync(context, result, Operations.Shutdown, this.ShutdownVirtualMachineFormComplete);
         }
 
+        [LuisIntent("ListRunbooks")]
+        public async Task ListRunbooksAsync(IDialogContext context, LuisResult result)
+        {
+            var accessToken = await context.GetAccessToken(resourceId.Value);
+            if (string.IsNullOrEmpty(accessToken))
+            {
+                return;
+            }
+
+            var subscriptionId = context.GetSubscriptionId();
+
+            var automationAccounts = await new AzureRepository().ListRunbooksAsync(accessToken, subscriptionId);
+
+            if (automationAccounts.Any())
+            {
+                var runbooksText = automationAccounts.Aggregate(
+                     string.Empty,
+                    (current, next) =>
+                    {
+                        return current += next.Runbooks.Aggregate(
+                            string.Empty,
+                            (currentRunbooks, nextRunbook) => 
+                            {
+                                return currentRunbooks += $"\n\r• {nextRunbook.RunbookName} ({next.AutomationAccountName})";

```

Comment:
If only one Automation account, there is no need to put the Automation account name next to it, we should put "listing all runbooks from _automationaccountname_:"